helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -99,6 +99,7 @@ controller:
     type: {{ .Values.ingressNginx.controller.service.type }}
     annotations:
     {{- .Values.ingressNginx.controller.service.annotations | nindent 6 }}
+    loadBalancerSourceRanges: {{- toYaml .Values.ingressNginx.controller.service.loadBalancerSourceRanges | nindent 6 }}
```

Comment:
Should this be wrapped in some sort of `if ...` or will this not break anything if you're not using octavia LB?

Reply:
Yes that would be a good idé. Should we add a another variable where you have to specify if it is supported? Hopefully tho it is just skipped if it is not supported.

Just a quick look and I see that aws and google supports this.

Reply:
The default in ingress-nginx seems to be an empty array when not used. And it should apply to any load balancer as it is not Octavia specific, though I'm not sure it applies to a service load balancer.

Reply:
Then I'm fine with leaving it as it is